src/stories/Home.stories.mdx

```diff
@@ -90,7 +90,7 @@ The library exposes a mock data object for testing purposes:
 ```ts
 import { mockData } from '@h5web/lib';
 
-const dataset = mockData.datasets['c8f60c29-aae2-11ea-84a9-b94ddd2ec9e8'];
+const dataset = mockData.datasets['c8f60c29-aae2-11ea-84a9-b94ddd2ec9e8']; // THIS WONT WORK
```

Comment:
This needs to be changed by something more robust. I am reluctant to put the method I used in the stories as it is a bit heavy.

Reply:
The way you find mock datasets is fine I think. Could put it in a utility function somewhere, perhaps in the mock data provider's folder and expose it in the library so it can be used in other stories and in the `Home` documentation?